content_scripts/hud.coffee

```diff
@@ -42,6 +56,28 @@ HUD =
     else
       @tween.fade 0, 150, => @hide true, updateIndicator
 
+  hideFindMode: (data) ->
+    # An element element won't receive a focus event if the search landed on it while we were in the HUD
+    # iframe. To end up with the correct modes active, we create a focus/blur event manually after refocusing
+    # this window.
+    window.focus()
+
+    sel = window.getSelection()
+    focusNode = if not sel.focusNode?
+      null
+    else if sel.focusNode == sel.anchorNode and sel.focusOffset == sel.anchorOffset
+      # The selection either *is* an element, or is inside an opaque element (eg. <input>).
+      sel.focusNode.childNodes[sel.focusOffset]
+    else if sel.focusNode.nodeType != sel.focusNode.ELEMENT_NODE
+      sel.focusNode.parentElement
+    else
+      sel.focusNode
+    document.activeElement?.blur()
+    focusNode?.focus()
+
+    findModeQuery.rawQuery = data.query
+    handlerStack.bubbleEvent "keydown", data.event
```

Comment:
This was the easiest, small-change way to pass keys that are detected in the iframe to the `FindMode` `keydown` handler. I plan to turn it into a simple function call when refactoring after (/if) this PR is merged.